app/services/Countries.scala

```diff
@@ -16,46 +16,32 @@
 
 package services
 
-import play.api.libs.json._
-import services.model.Country
-import utils.FileReader
+import connectors.CodeListConnector
+import models.codes.Country
+import play.api.i18n.Messages
 
 object Countries {
```

Comment:
Maybe it's a wrong idea (I did not check if it's really possible) but couldn't `Countries` changed to be a singleton class so that you can directly inject `CodeListConnector`, and maybe `Messages` too, so that you do not need to pass it everywhere?